src/OrbitGl/App.cpp

```diff
@@ -2163,8 +2163,8 @@ void OrbitApp::SetMaxLocalMarkerDepthPerCommandBuffer(
 }
 
 void OrbitApp::SelectFunction(const orbit_client_protos::FunctionInfo& func) {
-  LOG("Selected %s (address_=0x%" PRIx64 ", loaded_module_path_=%s)", func.pretty_name(),
-      func.address(), func.module_path());
+  LOG("Selected %s (address_=0x%x, loaded_module_path_=%s)", func.pretty_name(), func.address(),
```

Comment:
Nit: Instead of `0x%x`, abseil also offers `%#x` which automatically adds the needed prefix ("0x" in that case)

Reply:
Done! Thanks!